packages/react-components/src/templates/GroupProfileHeader.tsx

```diff
@@ -95,12 +99,12 @@ const GroupProfileHeader: React.FC<GroupProfileHeaderProps> = ({
             <TabLink
               href={route.upcoming({}).$ + queryParamString(searchQuery)}
             >
-              Upcoming Events
+              Upcoming Events{` (${upcomingEventsCount})`}
```

Comment:
Whats going on here? cant this just be {upcomingEventsCount}

Reply:
this is due to the space that needs to be displayed